code/modules/admin/_defines.dm

```diff
@@ -0,0 +1,13 @@
+#define TICKET_AUTORESPONSE_BEING_HANDLED "\blue <b>NOTICE: <font color=red>[usr.key]</font> is autoresponding with <font color='#009900'>'[choice]'</font>. The issue is already being dealt with.</b>"
+#define TICKET_AUTORESPONSE_FIXED "\blue <b>NOTICE: <font color=red>[usr.key]</font> is autoresponding with <font color='#009900'>'[choice]'</font>. The issue is already fixed.</b>"
+#define TICKET_AUTORESPONSE_THANKS "\blue <b>NOTICE: <font color=red>[usr.key]</font> is autoresponding with <font color='#009900'>'[choice]'</font>. Have a SCP day!</b>"
+#define TICKET_AUTORESPONSE_REQUESTING_TO_BE_SCP "\blue <b>NOTICE: <font color=red>[usr.key]</font> is autoresponding with <font color='#009900'>'[choice]'</font>. Only under certain circumstances will an admin put you into an SCP. Examples of these circumstances are: Imminent Test or gross negligence in containment. Do not repeatedly ask to become an SCP, as this might get you a mute.</b>"
+#define TICKET_AUTORESPONSE_BECOMING_AN_SCP "\blue <b>NOTICE: <font color=red>[usr.key]</font> is autoresponding with <font color='#009900'>'[choice]'</font>. In order to become an SCP, the alert level should be above red, and you should be dead for at least 5 minutes.</b>"
+#define TICKET_AUTORESPONSE_GITHUB_ISSUE "\blue <b>NOTICE: <font color=red>[usr.key]</font> is autoresponding with <font color='#009900'>'[choice]'</font>. Thanks for reporting this issue! Though, for maximum effectivity, please create an issue report on our <a href='https://github.com/Foundation-19/Foundation-19/issues'>GitHub</a>.</b>"
+#define TICKET_AUTORESPONSE_RUST_ENGINE "\blue <b>NOTICE: <font color=red>[usr.key]</font> is autoresponding with <font color='#009900'>'[choice]'</font>. The RUST Engine can be set up by finding the placed manual in the chamber area's, or alternatively you can visit <a href='https://bay.ss13.me/Guides/RUST'>the wiki guide on this engine</a>.</b>"
+#define TICKET_AUTORESPONSE_D_CLASS_RIOT "\blue <b>NOTICE: <font color=red>[usr.key]</font> is autoresponding with <font color='#009900'>'[choice]'</font>. D-Class may only riot with permission from a member of staff, and needs a good reason. If you wish to start a riot, you are to adminhelp again, outlining your reason. A member of staff will take your request under advisement.</b>"
+#define TICKET_AUTORESPONSE_D_CLASS_CELLS "\blue <b>NOTICE: <font color=red>[usr.key]</font> is autoresponding with <font color='#009900'>'[choice]'</font>. You can only get out if the on-duty guards feel like releasing you. It's best to wait and watch a YouTube video until such time.</b>"
+#define TICKET_AUTORESPONSE_DIRECTIONS_OR_LOST "\blue <b>NOTICE: <font color=red>[usr.key]</font> is autoresponding with <font color='#009900'>'[choice]'</font>. If you are lost, it is best to keep an eye on directional signs, ask a fellow player or find Holomaps, if they have been added to the current map.</b>"
```

Comment:
I feel like this should include a mentorhelp suggestion, since mentors can and should be helping with this sort of thing